cms/static/js/views/active_video_upload_list.js

```diff
@@ -169,6 +196,43 @@ define(
                 this.setStatus(data.cid, ActiveVideoUpload.STATUS_FAILED);
             },
 
+            hideErrorMessage: function() {
+                if (this.fileErrorMsg) {
+                    this.fileErrorMsg.hide();
+                }
+            },
+
+            showErrorMessage: function(error) {
+                this.fileErrorMsg = new NotificationView.Error({
+                    title: gettext('Your file could not be uploaded'),
+                    message: error
+                });
+                this.fileErrorMsg.show();
+            },
+
+            validateFile: function(data) {
+                var self = this,
+                    error = '',
+                    fileName,
+                    fileType;
+
+                $.each(data.files, function(index, file) {  // eslint-disable-line consistent-return
+                    fileName = file.name;
+                    fileType = fileName.substr(fileName.lastIndexOf('.') + 1);
+                    // validate file type
+                    if (!_.contains(self.videoSupportedFileFormats, fileType)) {
+                        error = gettext(
+                            '`{filename}` has unsupported file format. ' +
```

Comment:
@catong Can you review error message here. This is the message that is shown to user.